sdk/cosmos/azure-cosmos/azure/cosmos/container.py

```diff
@@ -488,7 +489,7 @@ def create_item(
         request_options = build_options(kwargs)
         response_hook = kwargs.pop('response_hook', None)
 
-        request_options["disableAutomaticIdGeneration"] = True
+        request_options["enableAutomaticIdGeneration"] = kwargs.pop('enable_automatic_id_generation', False)
```

Comment:
The request_options key here is a service value - not sure it can be changed like this?
I think you'll need something like:
```python
        request_options["disableAutomaticIdGeneration"] = not kwargs.pop('enable_automatic_id_generation', False)
```

Reply:
yeah definitely -  I can't push at the moment, having issues authenticating to github. I slapped something together and then was testing pushing. I'll reping this when ready for review

Reply:
Awesome, thanks! :) 
Good luck with the github trouble - let me know if you need me to make any updates from my end.